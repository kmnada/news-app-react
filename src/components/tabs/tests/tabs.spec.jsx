import React from 'react';
import { shallow } from 'enzyme';
import TabsComponent from '../tabs';

test('should render Tabs component', () => {
  const wrapper = shallow(<TabsComponent />);
  expect(wrapper).toMatchSnapshot();
});
