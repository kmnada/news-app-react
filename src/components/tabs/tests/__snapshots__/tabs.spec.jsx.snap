// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Tabs component 1`] = `ShallowWrapper {}`;
